src/spikeinterface/generation/drift_tools.py

```diff
@@ -40,9 +40,13 @@ def interpolate_templates(templates_array, source_locations, dest_locations, int
     source_locations = np.asarray(source_locations)
     dest_locations = np.asarray(dest_locations)
     if dest_locations.ndim == 2:
-        new_shape = templates_array.shape
+        new_shape = (*templates_array.shape[:2], len(dest_locations))
```

Comment:
using len() is always ambiguous I think for arrays.
len = dest_locations.shape[0] but here I think you wanted dest_locations.shape[1]
@cwindolf  : can you check this ? @alejoe91 merged this too quickly and I think we need a new PR now.
no sure.